### PR TITLE
Fixed #25324 -- Registered ModelAdmins with @admin.register decorator.

### PR DESCRIPTION
**Pull request by Y3K**
https://code.djangoproject.com/ticket/25324

### CODE REVIEW

File: django/contrib/auth/admin.py [hunk 1]
```diff
@@ -80,9 +82,11 @@ def get_form(self, request, obj=None, **kwargs):
         return super(UserAdmin, self).get_form(request, obj, **defaults)
 
     def get_urls(self):
-        return [
-            url(r'^(.+)/password/$', self.admin_site.admin_view(self.user_change_password), name='auth_user_password_change'),
-        ] + super(UserAdmin, self).get_urls()
+        return [url(
```
Comment (timgraham) [inline django/contrib/auth/admin.py]: 1. Please try to refrain from making unrelated cleanups like this (when otherwise not touching the code) as it's confusing with looking through commit history.
2. ModelClass instances -> ModelAdmins and Fix -> Fixed in the commit message
3. Thanks!

Comment (Y3K) [inline django/contrib/auth/admin.py]: @timgraham noted!

About the line... I just changed it because it was *really* long, above 119 characters (my _hard_ limit).

I'll revert that change and update replace `MovelClass` with `ModelAdmins` (sorry for this).

Thank you!

Comment (timgraham) [inline django/contrib/auth/admin.py]: I agree about limiting line length, just not as part of this PR. I've reopened https://code.djangoproject.com/ticket/23395.

**Comment by Y3K** [comment]
Sorry for the delay @timgraham 

I've addressed your comments now.

Thanks!

File: django/contrib/auth/admin.py [hunk 2]
```diff
@@ -176,7 +178,8 @@ def user_change_password(self, request, id, form_url=''):
 
         request.current_app = self.admin_site.name
 
-        return TemplateResponse(request,
+        return TemplateResponse(
+            request,
```
Comment (MarkusH) [inline django/contrib/auth/admin.py]: While you're around, can you use the following style:
```python
return TemplateResponse(
    request,
    self.change_user_password_template or '...',
    context,
)
```

Comment (MarkusH) [inline django/contrib/auth/admin.py]: Actually, since that's totally unrelated to the actual change, can you revert your change, please

Comment (timgraham) [inline django/contrib/auth/admin.py]: well, I'd suggest to revert this change entirely as it's unrelated to the patch.

**Comment by MarkusH** [comment]
LGTM

File: django/contrib/auth/admin.py (hunk 2 above)
Comment (Y3K) [inline django/contrib/auth/admin.py]: Oh bummer! I'm so used to move stuff around until all the warning are gone, haha.

I'll rever this change right now, thanks!

**Comment by Y3K** [comment]
@MarkusH @timgraham 

I've removed every not-related code change now, I'll avoid doing that again.

Thanks!

**Comment by MarkusH** [comment]
Merged in 235caabacc37278487d1107f6bbda90cc208cd64. Thanks! :watermelon:
